test/JuMP_sparse_linear_mpc.jl

```diff
@@ -0,0 +1,231 @@
+using Ipopt, JuMP, Random, LinearAlgebra
```

Comment:
some duplicate `using` here. Ideally, we want to `using` all package at the beginning of `runtests.jl` script

Reply:
and possibly rename the script to `sparse_lq_test.jl`